GUIDE.md

```diff
@@ -19,7 +19,7 @@ Please refer to [this section](/README.md/#minimum-hardware-requirements) for ha
 ## System Environment
  - Ubuntu 20.04+ (tested and verified)
  - (Optional) Docker 24.0.5+ (would simplify the process)
- - (Optional) Go 1.20.* (not compatible with Go 1.21 yet) and Node.js 16+ (can be installed following the [steps](#install-dependencies))
+ - (Optional) Go 1.21.* and Node.js 16+ (can be installed following the [steps](#install-dependencies))
```

Comment:
(Optional) Go 1.20+ and Node.js 16+ (can be installed following the [steps](#install-dependencies))